cpp/src/dictionary/detail/concatenate.cu

```diff
@@ -245,7 +245,8 @@ std::unique_ptr<column> concatenate(host_span<column_view const> columns,
   std::vector<column_view> indices_views(columns.size());
   std::transform(columns.begin(), columns.end(), indices_views.begin(), [](auto cv) {
     auto dict_view = dictionary_column_view(cv);
-    if (dict_view.is_empty()) return column_view{data_type{type_id::UINT32}, 0, nullptr};
+    if (dict_view.is_empty())
+      return column_view{data_type{type_id::UINT32}, 0, nullptr, nullptr, 0};
```

Comment:
I strongly prefer braces for readability in cases where conditionals break lines.
```suggestion
    if (dict_view.is_empty()) {
      return column_view{data_type{type_id::UINT32}, 0, nullptr, nullptr, 0};
    }
```